package.json

```diff
@@ -82,11 +85,15 @@
     "@langchain/openai": "0.4.4",
     "@octokit/rest": "21.1.1",
     "@slack/bolt": "4.2.0",
+    "@slack/web-api": "^7.8.0",
+    "axios": "1.7.9",
```

Comment:
please do not use `axios`. we are dependent on `adobe-fetch` and we have the `tracingFetch` method in `spacecat-shared-utils`